From ben@simtec.co.uk  Wed Mar 11 11:38:37 2009
From: Ben Dooks <ben@simtec.co.uk>
Date: Thu, 26 Feb 2009 23:03:15 +0000
Subject: USB: ohci-s3c2410: fix name of bus clock
Cc: dbrownell@users.sourceforge.net
Message-ID: <20090226230314.880253822@fluff.org.uk>


The USB bus clock is usb-bus-host, so print the correct name in the
dev_err() statement if we cannot find it.

Signed-off-by: Ben Dooks <ben@simtec.co.uk>
Acked-by: David Brownell <dbrownell@users.sourceforge.net>

---
 drivers/usb/host/ohci-s3c2410.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/host/ohci-s3c2410.c
+++ b/drivers/usb/host/ohci-s3c2410.c
@@ -371,7 +371,7 @@ static int usb_hcd_s3c2410_probe (const 
 
 	usb_clk = clk_get(&dev->dev, "usb-bus-host");
 	if (IS_ERR(usb_clk)) {
-		dev_err(&dev->dev, "cannot get usb-host clock\n");
+		dev_err(&dev->dev, "cannot get usb-bus-host clock\n");
 		retval = -ENOENT;
 		goto err_clk;
 	}
